optaplanner-quickstarts/quarkus-school-timetabling/src/main/java/org/acme/schooltimetabling/domain/Timeslot.java

```diff
@@ -54,6 +54,11 @@ public Timeslot(DayOfWeek dayOfWeek, LocalTime startTime, LocalTime endTime) {
         this.endTime = endTime;
     }
 
+    public Timeslot(long id, DayOfWeek dayOfWeek, LocalTime startTime) {
```

Comment:
In a real project, this constructor would deserve a documentation explaining how long a timeslot it creates if you do not provide an end time. Not sure if it's worth it for the quickstart. Your call.

Reply:
The end time doesn't really matter anyways; all that matter if time slots are exactly the same or not (the constraints do not check for overlap of time slots, but equality of time slots)

Reply:
(actually, it does matter for one constraint - teacherTimeEfficiency and studentGroupSubjectVariety, where we check for > 30 minutes gaps). There is an implict assumption that timeslots do not overlap

Reply:
Considering that this would be the only Javadoc in the entire project, I decided against it. Merging.